### DISCUSSION TITLE
DFG LUT and the cloth BRDF - which one is correct?

### DISCUSSION THREAD

**Discussion by ncuxonaT** [google/filament#5415] [Q&A]
Hi! I've ported all the code related to the DFG LUT creation process to GLSL trying to reproduce the LUT from the docs. But the cloth BRDF DG term I've got looks quite different. I've also ripped the LUT used in Filament web demos to compare with others. Could you please help me to figure out which one is correct?
Here is the LUT from the docs and it's blue channel (cloth DG):
![image](https://google.github.io/filament/images/ibl/dfg_cloth.png) ![docs_lut_blue](https://user-images.githubusercontent.com/15171368/162593661-2dbb1252-4048-4017-b302-47202bc9ef2c.png)
And here is the LUT I've got and the one from web demos (upscaled from 64x64):
![my_dg](https://user-images.githubusercontent.com/15171368/162593993-c4eaec59-a6e7-4c1f-9f4d-d6b90661b5d0.png) ![web_demo_dg](https://user-images.githubusercontent.com/15171368/162594083-68cc5a56-0c48-4b15-973e-94821cdcd418.png)

**Comment by z2ZTO0o** [comment]
@ncuxonaT Hi, I have calculated the same DFG LUT as you. Have you found the reason for the issue?
